packages/dashmate/src/listr/prompts/createIpAndPortsForm.js

```diff
@@ -59,19 +55,15 @@ function createIpAndPortsFormFactory(defaultConfigs) {
       return validatePort(value);
     }
 
-    let initialIp;
-    if (options.initialIp === null || options.initialIp === undefined) {
-      initialIp = await Promise.race([
-        publicIp.v4().catch(() => ''),
-        // Resolve in 10 seconds if public IP is not available
-        wait(10000).then(() => ''),
-      ]);
+    let { initialIp } = options;
+
+    if (!initialIp) {
```

Comment:
We should distinguish null/undefined and empty string

Reply:
Why? I thought it should detect in any cases (because user can easily put his own)